include/oneapi/dpl/pstl/utils.h

```diff
@@ -48,6 +48,16 @@ namespace dpl
 namespace __internal
 {
 
+enum class __algorithm_type
+{
+    transform_scan,
+    set_operation,
+    scan_copy,
+    partition,
+    unique,
+    other
```

Comment:
Looks like this enum value ```other``` not used in your patch in other places. Is this enum value really required?

Reply:
Also we might want to give elements explicit integer values. Just as an extra caution for future modifications.

Reply:
Good points for both. I will remove `other` and give explicit values for the enums.